apps/admin/src/features/group-management/model/mapper.ts

```
@@ -0,0 +1,91 @@
+    groupIntroduction: dto.description,
+    leader: dto.leader ? mapMemberCardDtoToMemberSummary(dto.leader) : undefined,
+    members: dto.members.map(mapMemberCardDtoToMemberSummary),
+  };
+};
+
+// memberIds에 팀장 id 추가
+const buildMemberIdsIncludingLeader = (draft: GroupFormDraft): number[] => {
+  const ids = new Set<number>();
+
+  if (draft.leader) ids.add(draft.leader.id);
+  draft.members.forEach((m) => ids.add(m.id));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)